tasks/tasks/search/bert_search.py

```diff
@@ -0,0 +1,170 @@
+# from __future__ import annotations
```

Comment:
I see you have this file duplicated here, and at `api/search/bert_search.py`. Seems like you'd probably want to delete one, and pull everything from the other one